source/data_tests/index.rst

```diff
@@ -4,176 +4,147 @@
-+-------------------------------------------+---------------------------------------+
-| :doc:`streamingSymbolsCheck`              | `streamingSymbolsCheck`_              |
-+-------------------------------------------+---------------------------------------+
-| :doc:`streamingTimeGrow`                  | `streamingTimeGrow`_                  |
-+-------------------------------------------+---------------------------------------+
-| :doc:`streamingTransferEncodingHeader`    | `streamingTransferEncodingHeader`_    |
-+-------------------------------------------+---------------------------------------+
-| :doc:`streamingUnauthorizedRequest`       | `streamingUnauthorizedRequest`_       |
-+-------------------------------------------+---------------------------------------+
-
-`/symbol_info`_ tests
-----------------
-
-+-----------------------------------------+-------------------------------------+
-| Test description                        | GituHb Link                         |
-+=========================================+=====================================+
-| :doc:`symbolsResponseFieldsCorrectness` | `symbolsResponseFieldsCorrectness`_ |
-+-----------------------------------------+-------------------------------------+
-| :doc:`symbolsUnauthorizedRequest`       | `symbolsUnauthorizedRequest`_       |
-+-----------------------------------------+-------------------------------------+
+  :depth: 1
+
+After you implement API, contact the TradingView team.
+They will prepare a test suite in Jenkins and provide you with credentials.
+The tests help the broker ensure that the integration with the TradingView user interface works correctly.
+
+Run tests
+...........
+
+1. Log into `Jenkins`_. You'll see the main page of your build plans.
+2. Select *Data_Integrations → inspect_<project_name>* to open your project page.
+3. Go to *Build with parameters* and select the parameters that correspond to the endpoints of your API. Here, you can select all endpoints at once or just one.
+
+  .. note:: 
+    The ``/history`` and ``/streaming`` tests require the ``/symbol`` and ``/groups`` (if you have groups implemented) tests to pass.
+    Otherwise, even if there are no issues with ``/history`` and ``/streaming``, the tests will fail.
+
+  .. image:: ../../images/DataTests_BuildWithParameters.png
+      :alt: Build with parameters
+      :align: center
+
+4. Select *Build*.
+5. Wait for the tests to complete. You can check the progress bar in *Build history*.
+6. Select the latest build number to view `test results <#test-results>`__.
+
+Check test results
+...................
+
+On the project's page → *Build History*, you can see the list of latest builds.
+Click on the last build number to see the results.
+
+.. image:: ../../images/DataTests_BuildHistory.png
+    :alt: Build history
+    :align: center
+
+
+The build number can be highlighted in one of the following colors:
```

Comment:
Nitpick:
To be completely accurate, we can say that `the build can be highlighted` or `the build is accompanied with an indicator`.